cloudbrain/rt_server/rt_server.py

```diff
@@ -14,6 +15,7 @@
 
 SERVER_PORT = 31415
 logging.getLogger().setLevel(logging.INFO)
+recursivedict = lambda: defaultdict(recursivedict)
```

Comment:
@flysonic10 I love the `recursivedict`: it's so functional! :smile: I have only just one concern about the fact that I'd rather :boom: in case of an unexpected lookup, instead of implicitly creating another dictionary for the missing key.
I was checking the `metric_exists` method and we actually perform some `has_key` checks; so my feeling is that it might be better to be explicit from now about when checking the keys of the `subscriber`s. That is also in perspective of adding future checks and communication between the RTServer and the API/OAuth layer to allow specific `device_id`s for users, specific metrics, etc.